src/Symfony/Component/Console/Application.php

```diff
@@ -615,10 +638,10 @@ public static function getAbbreviations($names)
     /**
      * Renders a caught exception.
      *
-     * @param \Exception      $e      An exception instance
-     * @param OutputInterface $output An OutputInterface instance
+     * @param \Throwable|\Exception $e      An throwable instance
+     * @param OutputInterface       $output An OutputInterface instance
      */
-    public function renderException(\Exception $e, OutputInterface $output)
+    public function renderException(/*\Throwable */$e, OutputInterface $output)
```

Comment:
Indeed. For now I've deprecated the `renderException` method in favor of `renderThrowable`.

The typehint is still commented and would be uncommented as soon as the PHP 5 support is dropped. I guess it's likely to happen for 4.0, so I can add it to `UPGRADE-4.0.md` file ?